app/views/layouts/application.html.slim

```diff
@@ -10,5 +10,6 @@ html
     p.notice
       = notice
     p.alert
-      = alert
-    = yield
+      - flash.each do |message_type, message|
```

Comment:
I see issue here with code ...
we have above.
p.notice
p.alert
and now we have flash.each

just leave flash.each in div. Remove "p"

Reply:
It is true. Now we haven't this issue